Pipelines/Templates/update-deployment-settings.yml

```diff
@@ -249,6 +261,38 @@ steps:
             Authorization = "Bearer $env:SYSTEM_ACCESSTOKEN"
         }
         $buildDefinitionResponseResults = $fullBuildDefinitionResponse.value
+        if($buildDefinitionResponseResults.length -ne $deploymentSteps.Count) {
```

Comment:
Will this handle if deployment have been deleted? How about if deployment step has been changed?

Reply:
The coe-cli is re-entrant so it should handle if some of the pipelines exist and others do. This does assume, as does the app for now that the setup is using the solution branching (v-next) strategy. Will verify